setup.py

```diff
@@ -46,6 +46,9 @@
     ]},
     scripts=['bin/copy_sphinxgallery.sh', 'bin/sphx_glr_python_to_jupyter.py'],
     url="https://sphinx-gallery.github.io",
+    project_urls={
+        "Source": "https://github.com/sphinx-gallery/sphinx-gallery",
```

Comment:
```suggestion
        "Source": "https://github.com/sphinx-gallery/sphinx-gallery",
        "Documentation": "https://sphinx-gallery.github.io",
```

Reply:
`url` already point to docs so I think it's duplication

Reply:
Indeed it's the same link -- but perhaps it's worth it to have both "Homepage" and "Documentation" show up in the sidebar?